opcua_tools/nodeset_generator.py

```diff
@@ -83,6 +101,18 @@ def generate_references_xml(nodes, references):
     references.loc[~references['target_same_ns'], 'NodeId'] = references.loc[~references['target_same_ns'], 'Src']
 
     # Outgoing reference when target is not in same nodeset
+    references.loc[~references['target_same_ns'], 'xml'] = "<Reference ReferenceType=\"{}\">{}</Reference>".format(
+        references.loc[~references['target_same_ns'], 'ReferenceType'],
+        references.loc[~references['target_same_ns'], 'Trg'])
+
+    # Incoming reference only when target is in same nodeset to be generated
+    references.loc[references['target_same_ns'], 'xml'] = "<Reference ReferenceType=\"{}\" IsForward=\"false\">{}</Reference>".format(
+        references.loc[references['target_same_ns'], 'ReferenceType'],
+        references.loc[references['target_same_ns'], 'Src'])

```

Comment:
Same problem as above, are you sure this works?

Reply:
Reverted - thanks